### PR TITLE
added h1 tags

### PR DESCRIPTION
**Pull request by kgds-dev**
Added test h1 tags

### CODE REVIEW

**Review by matiLopezJay — APPROVED** [review]
looks good again

**Review by JohnJayvee — APPROVED** [review]
Pogi ni sir dha

**Comment by dharylovey** [comment]
all goods

**Review by JohnJayvee — CHANGES_REQUESTED** [review]
request

File: index.html [hunk 1]
```diff
@@ -14,6 +14,15 @@
...
     <p>Kimbo6</p>
+
+    <h1>KODEGO</h1>
+    <h1>KODEGO</h1>
+    <h1>KODEGO</h1>
+    <h1>KODEGO</h1>
+    <h1>KODEGO</h1>
+    <h1>KODEGO</h1>
+    <h1>KODEGO</h1>
+    <h1>KODEGO</h1>
```
Comment (matiLopezJay) [inline index.html]: use h2 tag
```suggestion
    <h2>KODEGO</h2>
```

File: index.html [hunk 2]
```diff
@@ -14,6 +14,15 @@
     <p>Kimbo4</p>
     <p>Kimbo5</p>
     <p>Kimbo6</p>
+
+    <h1>KODEGO</h1>
+    <h1>KODEGO</h1>
+    <h1>KODEGO</h1>
```
Comment (JohnJayvee) [inline index.html]: galing naman

**Review by dharylovey — APPROVED** [review]

**Review by dharylovey — APPROVED** [review]

**Review by JohnJayvee — CHANGES_REQUESTED** [review]
galing naman

File: index.html (hunk 2 above)
Comment (kgds-dev) [reply index.html]: sige palitan ko to

**Review by matiLopezJay — CHANGES_REQUESTED** [review]

**Review by JohnJayvee — APPROVED** [review]
ayos

**Pull request merged**